src/pcapi/core/offers/api.py

```diff
@@ -263,6 +264,10 @@ def as_utc_without_timezone(d: datetime.datetime) -> datetime.datetime:
         bookings = bookings_repository.find_not_cancelled_bookings_by_stock(stock)
         if bookings:
             bookings = update_confirmation_dates(bookings, beginning)
+            date_in_two_days = datetime.datetime.now() + datetime.timedelta(days=2)
+            check_event_is_in_more_than_48_hours = beginning > as_utc_without_timezone(date_in_two_days)
```

Comment:
La fonction `as_utc_without_timezone()` n'étant pas là pour rester (enfin j'espère !), je proposerais plutôt d'utiliser directement `datetime.utcnow()` :

```suggestion
            date_in_two_days = datetime.datetime.utcnow() + datetime.timedelta(days=2)
            check_event_is_in_more_than_48_hours = beginning > date_in_two_days
```